src/components/Proposals/DisputableActionStatus.js

```diff
@@ -0,0 +1,147 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import {
+  Box,
+  GU,
+  IconLock,
+  Info,
+  Link,
+  textStyle,
+  TokenAmount,
+  useTheme,
+} from '@aragon/ui'
+import {
+  DISPUTABLE_VOTE_STATUSES,
+  VOTE_STATUS_PAUSED,
+} from '../../utils/disputable-vote-statuses'
+import { toMs } from '../../utils/date-utils'
+import DisputableActions from './DisputableActions'
+import DisputablePeriod from './DisputablePeriod'
+import DisputableStatusLabel from './DisputableStatusLabel'
+
+function hasDispute(vote) {
+  // TODO: Check if vote has dispute
+  return false
+}
+
+function DisputableActionStatus({ vote }) {
+  // TODO: get agreement title, replace  tokenAddress for tokenId
+
+  const { actionAmount } = vote.collateral
+  const { decimals, symbol } = vote.token
+
+  const tokenAddress = '0x6B175474E89094C44Da98b954EedeAC495271d0F'
+
+  const disputableStatus = DISPUTABLE_VOTE_STATUSES.get(vote.status)
+  const challenged = disputableStatus === VOTE_STATUS_PAUSED
+
+  return (
+    <Box heading="Disputable Action Status">
+      <ul>
+        <Item label="Status">
+          {disputableStatus && (
+            <DisputableStatusLabel status={disputableStatus} />
+          )}
+        </Item>
+        <Item label="Action collateral locked">
+          <div
+            css={`
+              display: flex;
+              align-items: center;
+            `}
+          >
+            <TokenAmount
+              address={tokenAddress}
+              amount={actionAmount}
+              decimals={decimals}
+              symbol={symbol}
+            />
+
+            <span
+              css={`
+                display: inline-flex;
+                padding-left: ${1 * GU}px;
+              `}
+            >
+              <IconLock size="small" />
+            </span>
+          </div>
+        </Item>
+        <Item label={challenged ? 'Settlement period' : 'Challenge period'}>
+          <DisputablePeriod
+            endDate={
+              challenged
+                ? toMs(vote.pausedAt)
+                : new Date(toMs(vote.endDate)).getTime()
```

Comment:
```suggestion
                : toMs(vote.endDate)
```

Would this work?

Also wondering, is this part of the connect API going to change in the future such that we don't need the `toMs()`?

Reply:
Yes :) F